protocol/messenger.go

```diff
@@ -1328,6 +1336,13 @@ func (m *Messenger) SendContactUpdates(ctx context.Context, ensName, profileImag
 	m.mutex.Lock()
 	defer m.mutex.Unlock()
 
+	myID := types.EncodeHex(crypto.FromECDSAPub(&m.identity.PublicKey))
```

Comment:
Maybe there should be a helper function called `contactIDFromPublicKey` or something? It is repeated a few times in this PR.